src/main/java/frc/robot/commands/TurnAuto.java

```diff
@@ -53,7 +53,7 @@ public void execute() {
       SmartDashboard.putNumber("TurnAngle", RobotContainer.m_gyroSubsystem.getAngle());
       System.out.println("Execute Works");
```

Comment:
Remove lines like this from code that is going into Dev or Main.  If we need data output it should get logged so it gets saved.